deploy/examples/multus-validation-test-scc.yaml

```diff
@@ -0,0 +1,38 @@
+# ServiceAccount and RBAC to support running multus validation test on OpenShift
```

Comment:
This seems more like rbac than scc. How about naming the file `multus-validation-test-rbac.yaml`?

Reply:
I thought SCC made sense because the RBAC gives access to an SCC, and SCCs are only available on openshift, implying this is an openshift-focused manifest.